### ISSUE TITLE
Bug with admin AJAX requests

### ISSUE THREAD

**Issue by chrisvanpatten** [kellenmace/facebook-conversion-pixel#2]
Version 1.3.3 introduced a fix for admin-ajax requests, but it doesn't cover every case. For instance, updating WooCommerce Subscriptions from the 1.x to 2.x branch fails while running an automated update procedure depending on admin-ajax, because the $_GET['page'] variable isn't set, which causes an error:

```
PHP Notice:  Undefined index: page in /home/flintrehab/dev.flintrehab.com/current/public/content/plugins/facebook-conversion-pixel/includes/admin.php on line 59
```

Should be a simple change, just switching the checks for 'fb_pxl_options' to this:

```php
if ( ! isset( $_GET['page'] ) || 'fb_pxl_options' !== $_GET['page'] ) {
...
```

I've filed a pull request (#1) to sync this repo with the WP.org version. Once that's set, I can provide another PR to fix this issue!

**Comment by chrisvanpatten** [comment]
I've got this working over in this branch: https://github.com/chrisvanpatten/facebook-conversion-pixel/tree/fix_ajax_calls

Once #1 is merged, I can file a PR for this.

**Comment by kellenmace** [comment]
@chrisvanpatten - I've merged the changes for #1 to sync this repo up with WP.org. Can you please submit a pull request for #2? Then I'll gladly merge that, as well. Thanks a lot for identifying and fixing this bug!
